CHANGELOG.md

```diff
@@ -34,6 +34,20 @@ Ref: https://keepachangelog.com/en/1.0.0/
 -->
 
 # Changelog
+
+## v25.18.0
+
+- 10b84b4c Fix sqsdomain package version (#508)
+- 9de14897 SQS-347 | integration, synthetic monitoring tests for active orders (#504)
+- 828fbb59 ci: disable code rabbit for tests (#505)
```

Comment:
We should only track client-facing changes in the changelog and can remove Ci or tests